### ISSUE TITLE
Feature Request for Educators and Language Teachers

### ISSUE THREAD

**Issue by inglesuniversal** [jitsi/jitsi-meet#7151]
Please add automatic subtitles to live video conference as current user speaks.

and two language subtitles in two custom colors would be even better. One row in English and one row in Spanish (as an example)

Check: meet.google.com

**Comment by darthvader2** [comment]
i will try to do it.

anybody already working on it ?

**Comment by Echolon** [comment]
@darthvader2 please check previous GSoC project first.

**Comment by Echolon** [comment]
@inglesuniversal can you please provide a proper title and fill the issue template?

**Comment by inglesuniversal** [comment]
Do I have to open a new issue for that or have a sample template available? I've never done it like that. I'm eager to help in any way possible. Thanks and STAY SAFE!!!

**Comment by jhiswin** [comment]
Relevant: https://nikvaessen.github.io/jekyll/update/2017/08/24/gsoc2017_work_product_submission.html

@Echolon I haven't looked deeply into it, but it looks like most of the work is already merged into Jitsi main itself?  Does that mean it's really just a matter of enabling it?

**Comment by inglesuniversal** [comment]
As an English as a Second Language certified instructor for the past 23 years. I must add that I tried meet.google.com this evening and the captioning worked really well and it dramatically improved the teaching experience online. But as anything new, it has big flaws. You cannot turn on or off easily, chrome starts jiggling the windows sizes as you move the mouse around, if I start speaking into my native language (Spanish) the words get mixed into the subtitles messing up the structure of the sentence I said in English. It would be amazingly better to split the translation into different lines and custom colors as each participant speaks or switch to another language. If I can further help you improve this you can count on me. I am a big fun of your project. Best regards

**Comment by jhiswin** [comment]
@inglesuniversal After taking more of a look into it, it looks like everything you need for transcription features is already merged into the mainline distros.  There's a repo with full deployment installer here: https://github.com/switnet-ltd/quick-jibri-installer

Main installation script for the Jigasi transcription portion:
https://github.com/switnet-ltd/quick-jibri-installer/blob/master/jigasi.sh

I'm not sure about the chances of this making it onto meet.jit.si though, if that's what you're hoping for.  Costs would probably be prohibitively high.  And I don't see any way to connect it to your own self-hosted Jigasi.

**Comment by kkarhan** [comment]
Seriously stalebot is value-removing and as @inglesuniversal hinted, closed-captions & subtitles are quite essential.

So [please](https://github.com/jitsi/jitsi-meet/issues/10931#issuecomment-2012029285) add this [essential accessibility feature](https://dair-community.social/@tomstoneham/112132440737732711) if it isn't there already...

**Comment by saghul** [comment]
Subtitles have been available for years now. I guess you didn't even bother checking eh?
